docs/index.md

```diff
@@ -33,3 +34,14 @@ provider "onepassword" {
 - `service_account_token` (String) A valid token for your 1Password Service Account. Can also be sourced from OP_SERVICE_ACCOUNT_TOKEN. Either this or `token` must be set.
 - `token` (String) A valid token for your 1Password Connect API. Can also be sourced from OP_CONNECT_TOKEN. Either this or `service_account_token` must be set.
 - `url` (String) The HTTP(S) URL where your 1Password Connect API can be found. Must be provided through the OP_CONNECT_HOST environment variable if this attribute is not set. Can be omitted, if service_account_token is set.
+
+## Known Service Accounts limitation:
+Users may encounter the following error `op error: (409) Conflict: Internal server conflict` when create/update/delete a bunch of items in the same vault as Terraform Provider handles each resource separately and therefore it makes a bunch of parallel requests using CLI for each of the resources.
```

Comment:
`[nitpick]`

Some phrasing improvements:
```suggestion
Users may encounter the following error `op error: (409) Conflict: Internal server conflict` when creating/updating/deleting multiple items in the same vault as Terraform handles each resource separately and therefore it makes parallel requests using the CLI for each resource.
```

Reply:
Thank you for the suggestion! I like the way @armstrongl represents it 😃